src/tools/miri/src/borrow_tracker/tree_borrows/tree.rs

```diff
@@ -704,7 +704,7 @@ impl Tree {
     }
 
     /// Traverses the entire tree looking for useless tags.
-    /// Returns true iff the tag it was called on is still live or has live children,
```

Comment:
issue: no, "iff" is short for "if and only if"

Reply:
Something I didn't know, thanks for the info! (Fixed)